context/app/static/js/components/detailPage/Attribution/Attribution.tsx

```diff
@@ -1,32 +1,95 @@
 import React from 'react';
 
+import Stack from '@mui/material/Stack';
+
 import { useFlaskDataContext } from 'js/components/Contexts';
 import { DetailPageSection } from 'js/components/detailPage/style';
 import EmailIconLink from 'js/shared-styles/Links/iconLinks/EmailIconLink';
 import SectionHeader from 'js/shared-styles/sections/SectionHeader';
-
-import { FlexPaper } from './style';
-import SectionItem from '../SectionItem';
+import SummaryPaper from 'js/shared-styles/sections/SectionPaper';
+import LabelledSectionText from 'js/shared-styles/sections/LabelledSectionText';
+import ContactUsLink from 'js/shared-styles/Links/ContactUsLink';
+import InfoAlert from 'js/shared-styles/alerts/InfoAlert';
 
 function Attribution() {
   const {
-    entity: { group_name, created_by_user_displayname, created_by_user_email, entity_type },
+    entity: {
+      group_name,
+      created_by_user_displayname,
+      created_by_user_email,
+      entity_type,
+      processing,
+      creation_action,
+      descendants,
+    },
   } = useFlaskDataContext();
 
-  return (
-    <DetailPageSection id="attribution">
-      <SectionHeader iconTooltipText={`Information about the group registering this ${entity_type?.toLowerCase()}.`}>
-        Attribution
-      </SectionHeader>
-      <FlexPaper>
-        <SectionItem label="Group">{group_name}</SectionItem>
-        <SectionItem label="Registered by" ml>
+  const isProcessedDataset = entity_type === 'Dataset' && processing === 'processed';
+  const isVisLiftedDataset = descendants?.find((descendant) => descendant.dataset_type === 'Histology [Image Pyramid]');
+  const isHiveProcessedDataset = isProcessedDataset && creation_action === 'Central Process';
+  const isSupportDataset = entity_type === 'Support';
+
+  const showContactAndAlert = isHiveProcessedDataset ?? isSupportDataset;
+  const showRegisteredBy = !(isProcessedDataset ?? isVisLiftedDataset ?? isHiveProcessedDataset ?? isSupportDataset);
+
+  const tooltips = {
+    group: 'This is the group that provided the raw dataset.',
+    contact: 'This is the contact for this data.',
+  };
+
+  const hiveInfoAlertText = `The data provided by the ${group_name} Group was centrally processed by HuBMAP. The results of this processing are independent of analyses conducted by the data providers or third parties.`;
+
+  const sections: {
+    label: string;
+    text: React.ReactNode;
+    tooltip?: string;
+  }[] = [
+    {
+      label: 'Group',
+      text: group_name,
```

Comment:
I would consider renaming the `text` prop to `children`, which would allow the JSX below to be simplified from:

```jsx
{sections.map(({ label, text, tooltip }) => (
	<LabelledSectionText key={label} label={label} iconTooltipText={tooltip}>
		{text}
	</LabelledSectionText>
))}
```

to:

```jsx
{sections.map((props) => <LabelledSectionText key={props.label} {...props} />))}
```